test/models/member_test.rb

```diff
@@ -260,6 +260,8 @@ class MemberTest < ActiveSupport::TestCase
   test "can destroy member with renewals" do
     loan = create(:overdue_loan)
     renew_loan(loan)
-    loan.member.destroy
+    assert_nothing_raised {
```

Comment:
I would have assumed Standard would force these to be do/ends but if it's cool with it then I am too 👍

Reply:
Yeah I honestly thought it would just fix them to whatever they were supposed to be 😂 

Looks like they fought the good fight on trying to enforce "semantic blocks" and eventually gave up and loosened the rules: https://github.com/standardrb/standard/pull/263

I think we can probably adopt the "single line {}, multi line do/end" convention, so I'll swap these to do/end.